firstvoices/backend/serializers/media_detail_serializers.py

```diff
@@ -95,6 +96,11 @@ class Meta(AudioSerializer.Meta):
         fields = AudioSerializer.Meta.fields + BaseUsageFieldSerializer.Meta.fields
 
 
+class DocumentDetailSerializer(DocumentSerializer):
+    class Meta(DocumentSerializer.Meta):
+        fields = DocumentSerializer.Meta.fields
```

Comment:
If there is no change from the parent serializer, there is no need to add this extra serializer. BUT, I think we should keep this and add the fields from the BaseUsageFieldSerializer

Reply:
For sure. Will add the BaseUsageFieldSerializer with the related_documents ticket 👍

Reply:
Which I realize will only work once you do the related ticket... so this is good as-is for now :)